fluvio-future-derive/Cargo.toml

```diff
@@ -1,12 +1,13 @@
 [package]
-name = "fluvio-test-derive"
-version = "0.1.1"
-edition = "2018"
+name = "fluvio-future-derive"
+version = "0.1.0"
+edition = "2021"
```

Comment:
is there reason this go back to 0.1.0? shouldn't this be 0.1.11 or 0.2

Reply:
I changed the crate name. I added a main proc macro to it, so its not only test macros anymore.

Reply:
got it